snapcraft/tests/test_lifecycle.py

```diff
@@ -1061,3 +1062,14 @@ def _create_core_snap(self):
             print('description: description', file=f)
 
         return lifecycle.snap(self.project_options, directory=core_path)
+
+
+class SnapErrorsTestCase(BaseLifecycleTestCase):
+
+    def test_mksquashfs_missing(self):
+        with mock.patch('shutil.which') as which_mock:
```

Comment:
in this case I just felt like it, if the test were more complex I'd have more of a reason to